cmd/launch.go

```diff
@@ -67,6 +67,8 @@ func init() {
 		"The absolute filepath to a bash script passed to the instance and executed after the instance starts (user data)")
 	launchCmd.Flags().StringToStringVar(&flagConfig.UserTags, "tags", nil,
 		"The tags applied to instances and volumes at launch (Example: tag1=val1,tag2=val2)")
+	launchCmd.Flags().StringVar(&flagConfig.CapacityType, "capacity-type", "",
```

Comment:
Can we update the README, particularly the sections for the available flags and the example outputs (they don't mention capacity type)?

Reply:
Yes, good catch. I'll do that.

Reply:
Fixed in 40a75e8d1e38755eecf5f99b1eeab0d4233fd878